src/script/interpreter.cpp

```diff
@@ -60,6 +60,38 @@ static inline void popstack(std::vector<valtype>& stack)
     stack.pop_back();
 }
 
+static inline void pushasset(std::vector<valtype>& stack, const CConfidentialAsset& asset)
+{
+    valtype vchinpAsset;
+    vchinpAsset.insert(vchinpAsset.begin(), asset.vchCommitment.begin() + 1, asset.vchCommitment.begin() + 33);
+    valtype vchAssetPref;
+    vchAssetPref.insert(vchAssetPref.begin(), asset.vchCommitment.begin(), asset.vchCommitment.begin() + 1);
+    stack.push_back(vchinpAsset);
+    stack.push_back(vchAssetPref);
+}
+
+// Review: Not sure if this should inline or not? Leave it to the compiler??
```

Comment:
Might as well inline both of these for consistency's sake. But it really doesn't matter.